Description: Use data from unicode-data instead of fetching it from the net
Author: Sebastian Ramacher <sramacher@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-20

--- a/setup.py
+++ b/setup.py
@@ -51,8 +51,8 @@
     UCD_URL = ('http://www.unicode.org/Public/UNIDATA/extracted/'
                'DerivedGeneralCategory.txt')
 
-    EAW_IN = os.path.join(HERE, 'data', 'EastAsianWidth.txt')
-    UCD_IN = os.path.join(HERE, 'data', 'DerivedGeneralCategory.txt')
+    EAW_IN = '/usr/share/unicode/EastAsianWidth.txt'
+    UCD_IN = '/usr/share/unicode/extracted/DerivedGeneralCategory.txt'
 
     EAW_OUT = os.path.join(HERE, 'wcwidth', 'table_wide.py')
     ZERO_OUT = os.path.join(HERE, 'wcwidth', 'table_zero.py')
@@ -113,7 +113,6 @@
 
     def _do_east_asian(self):
         """Fetch and update east-asian tables."""
-        self._do_retrieve(self.EAW_URL, self.EAW_IN)
         (version, date, values) = self._parse_east_asian(
             fname=self.EAW_IN,
             properties=(u'W', u'F',)
@@ -123,7 +122,6 @@
 
     def _do_zero_width(self):
         """Fetch and update zero width tables."""
-        self._do_retrieve(self.UCD_URL, self.UCD_IN)
         (version, date, values) = self._parse_category(
             fname=self.UCD_IN,
             categories=('Me', 'Mn',)
